lib/Jsrt/CMakeLists.txt

```diff
@@ -52,3 +52,25 @@ target_include_directories (
     ../Runtime/Debug
     ../Parser
     )
+
+
+add_custom_target(ChakraCoreHeaders ALL
```

Comment:
We need to have this under `if(NOT CC_XCODE_PROJECT)` otherwise xcode build would break.